### DISCUSSION TITLE
Drop-down animation for menus

### DISCUSSION THREAD

**Discussion by Secret-chest** [linuxmint/discussions#25] [Ideas]
Before Cinnammon 5.6, all menus had a very nice but subtle drop-down animation. Now it's gone. Can I get it back now? I really miss it.

**Comment by kacperpaczos** [comment]
Can you show the earlier way of working?

**Reply by Secret-chest** [reply]
I can't sadly.

**Reply by kacperpaczos** [reply]
> I can't sadly.

I found the relevant thread on the bugtracker:
https://github.com/linuxmint/cinnamon/issues/10983

It should be working, but I don't see it working properly.
Cinnamon 5.6.8
![image](https://github.com/linuxmint/discussions/assets/23406555/6bb9dad8-c288-41bf-b32f-61b8c6b11611)
